### PR TITLE
Ncnews 10

### PR DESCRIPTION
**Pull request by baurzhaan**
please review my changes. Thanks.

### CODE REVIEW

**Review by k-huggs — COMMENTED** [review]
There's still some work to be done on this!

File: __tests__/news.test.js [hunk 1]
```diff
@@ -224,4 +224,58 @@ describe('9. GET /api/articles/:article_id/comments', () => {
...
+});
+
+describe('10. POST /api/articles/:article_id/comments', () => {
+  
+  test('returns an object if an input is an object', () => {  
+    return request(app)
+      .post('/api/articles/3/comments')
+      .send({ username: 'icellusedkars', body: 'just a comment' })
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
```
[the remark this answers is not in the document]
Comment (baurzhaan) [reply __tests__/news.test.js:236]: done

File: models/comments.models.js [hunk 2]
```diff
@@ -7,4 +7,29 @@ exports.selectCommentsByArticleId = (articleId) => {
...
+exports.insertCommentByArticleId = (articleId, comment) => {
+  const dateObject = new Date(Date.now());
+  const sqlQuery = 'INSERT INTO comments (body, article_id, author, votes, created_at) VALUES ($1, $2, $3, $4, $5) RETURNING *';
+  return db.query(sqlQuery, [comment.body, +articleId, comment.username, 0, dateObject.toISOString()])
+  .then(( {rows: insertedComment} ) => {
+    if (insertedComment.length > 0) {
+      insertedComment[0].created_at = insertedComment[0].created_at.getTime();
+      return insertedComment[0];
+    };
+    return Promise.reject({ code: 404, msg: 'Not foundAAA' });
```
[the remark this answers is not in the document]
Comment (baurzhaan) [reply models/comments.models.js:21]: sorry, was testing different behaviours while catching the errors )). Changed to the standard form of "Not found".

File: __tests__/news.test.js [hunk 3]
```diff
@@ -224,4 +224,58 @@ describe('9. GET /api/articles/:article_id/comments', () => {
...
+  
+  test('404: responds with message \'Not found\' when the article with article_id doesn\'t exist', () => {
+    return request(app)
+      .post('/api/articles/777/comments')
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('Not found');
+      });
+  });
+
```
[the remark this answers is not in the document]
Comment (baurzhaan) [reply __tests__/news.test.js:260]: added new tests

File: __tests__/news.test.js [hunk 4]
```diff
@@ -224,4 +224,58 @@ describe('9. GET /api/articles/:article_id/comments', () => {
...
+            article_id: 3,
+            author: 'butter_bridge',
+            votes: 0,
+            created_at: expect.any(Number) // current time
+          }
+        );
+      });
+  });
+  
+  test('404: responds with message \'Not found\' when the article with article_id doesn\'t exist', () => {
```
[the remark this answers is not in the document]
Comment (baurzhaan) [reply __tests__/news.test.js]: changed some tests

**Pull request merged**